discovery/entry.bash

```diff
@@ -2,5 +2,28 @@
 
 #start lldp daemon
 lldpd -dd &
+mv -f /usr/bin/ipmitool /usr/bin/ipmitool.real
+cat > /usr/bin/ipmitool <<EOF
+#!/bin/bash
+OUTPUT=\$(mktemp)
+/usr/bin/ipmitool.real \$@ > \$OUTPUT
+REALEXIT=\$?
+EXIT=\$REALEXIT
+if [ \$REALEXIT -ne 0 ]; then
+        grep -A1 "Mezz Slot" \$OUTPUT | grep -q "Device not present"
+        if [ \$? -eq 0 ]; then
+                EXIT=0
+        fi
+        grep -A1 "FRU Device Description" \$OUTPUT | grep -q "Device not present"
```

Comment:
Is it possible that there exist both "Device not present" and other errors?

Reply:
This checks for Device not present immediately after the FRU Device Description.  I have not found any examples or documentation for another error being displayed after, so it is an educated guess here.